cmd/testops/env/create/create.go

```diff
@@ -0,0 +1,83 @@
+package create
+
+import (
+	"fmt"
+	"github.com/qase-tms/qasectl/cmd/flags"
+	"github.com/qase-tms/qasectl/internal/client"
+	"github.com/qase-tms/qasectl/internal/service/env"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"log/slog"
+	"os"
+	"path"
+)
+
+const (
+	titleFlag       = "title"
+	descriptionFlag = "description"
+	slugFlag        = "slug"
+	hostFlag        = "host"
+	outputFlag      = "output"
+)
+
+// Command returns a new cobra command for create environments
+func Command() *cobra.Command {
+	var (
+		title       string
+		description string
+		slug        string
+		host        string
+		output      string
+	)
+
+	cmd := &cobra.Command{
+		Use:     "create",
+		Short:   "Create a new environment",
+		Example: "qli testops env create --title 'New environment' --description 'This is a environment' --slug local --host app.server.com --project 'PRJ' --token 'TOKEN'",
```

Comment:
Title and slug are required, let's put them first.

```suggestion
		Example: "qli testops env create --slug stage --title 'Stage environment' --description 'Testing code before production'  --host app.server.com --project 'PRJ' --token 'TOKEN'",
```

Reply:
fixed